README.md

```diff
@@ -88,6 +90,8 @@ pnpm preview
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/lucioaimar"><img src="https://avatars.githubusercontent.com/u/64326713?v=4?s=100" width="100px;" alt="Lucio Aimar"/><br /><sub><b>Lucio Aimar</b></sub></a><br /><a href="https://github.com/santoshyadavdev/angular-snippets/commits?author=lucioaimar" title="Documentation">📖</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/deepakrudrapaul"><img src="https://avatars.githubusercontent.com/u/25549935?v=4?s=100" width="100px;" alt="Deepak Rudra Paul"/><br /><sub><b>Deepak Rudra Paul</b></sub></a><br /><a href="https://github.com/santoshyadavdev/angular-snippets/commits?author=deepakrudrapaul" title="Documentation">📖</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/rubenperegrina"><img src="https://avatars.githubusercontent.com/u/23550574?v=4?s=100" width="100px;" alt="Rubén Peregrina"/><br /><sub><b>Rubén Peregrina</b></sub></a><br /><a href="https://github.com/santoshyadavdev/angular-snippets/commits?author=rubenperegrina" title="Documentation">📖</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/pawanpatil08"><img src="https://avatars.githubusercontent.com/u/16464034?v=4?s=100" width="100px;" alt="Pawan Patil"/><br /><sub><b>Pawan Patil</b></sub></a><br /><a href="https://github.com/santoshyadavdev/angular-snippets/commits?author=pawanpatil08" title="Code">💻</a></td>
+      
```

Comment:
This part should be auto-generated from `.all-contributorsrc`, so this shouldn't be manually committed.